policy/release/olm/olm.rego

```diff
@@ -318,6 +317,29 @@ deny contains result if {
 	result := lib.result_helper_with_term(rego.metadata.chain(), [img_str], img.ref.repo)
 }
 
+# METADATA
+# title: OLM bundle images are not multi-arch
+# description: >-
+#   OLM bundle images should be multi-arch. It should not be an OCI image index
+#   nor should it be a Docker v2s2 manifest list.
+# custom:
+#   short_name: olm_bundle_multi_arch
+#   failure_msg: The %q bundle image is a multi-arch reference.
+#   solution: >-
+#     Rebuild your bundle image without creating an image index.
+#   collections:
+#   - redhat
+#   effective_on: 2025-5-01T00:00:00Z
+deny contains result if {
+	# Parse manifests from snapshot
+	some csv_manifest in _csv_manifests
```

Comment:
I think you need this, since otherwise there's no other test making sure csv_manifest is not empty.

```suggestion
	count(csv_manifest) > 0
```

Reply:
Actually that's wrong. Disregard!

Reply:
The some implies it can't be empty.